python/pysrc/kaskada/_timestream.py

```diff
@@ -239,6 +239,37 @@ def __radd__(self, lhs: Union[Timestream, Literal]) -> Timestream:
             lhs = Timestream._literal(lhs, self._ffi_expr.session())
         return lhs.add(self)
 
+    def ceil(self) -> Timestream:
+        """
+        Create a Timestream for the number rounded up to the next largest integer.
+
+        Returns
+        -------
+        Timestream
+            The Timestream resulting from the numeric ceiling of this.
+        """
+        return Timestream._call("ceil", self)
+
+    def clamp(
+        self, min: Union[Literal, None] = None, max: Union[Literal, None] = None
```

Comment:
Man and max can also be timestreams, allowing for instance:

```
withdrawl = requested_withdrawl.clamp(max=Account.balance)
```

(eg., use the account balance as an upper bound on the withdrawl)